bundles/binding/org.openhab.binding.fritzboxtr064/README.md

```diff
@@ -120,12 +120,15 @@ Create file `transform/presence.map` and add:
 ```
 1=present
```

Comment:
I thisnk you should remove the 1 and 0 lines while you're at it, these values are not used.